app/geant-exporter/PhysicsList.cc

```diff
@@ -72,13 +73,15 @@ PhysicsList::~PhysicsList() = default;
  * triton, He3, alpha, and generic ion, along with Geant4's pseudo-particles
  * geantino and charged geantino.
  *
- * Currently only instantiating e+, e-, and gamma, as in Celeritas.
+ * Currently only instantiating e+, e-, gamma, and proton (the latter is needed
```

Comment:
Does this mean that we also store lambda tables for protons?  (then how about pions?).  Since we are not dealing with any hadron for now, should we ignore them for now - even though the current MSC model will handle cases for charged hadrons too (unless we simplified the curretn Geant4 code further).

Reply:
> Does this mean that we also store lambda tables for protons?

Not really. We still skip processes that are not from `e+`, `e-`, or `gamma` (see snippet below). I initialize `protons` to avoid a warning from the `msc` process class, which looks up for production cut couples (and at that point, if proton is not initialized, it will initialize it, and issue a warning message).

https://github.com/celeritas-project/celeritas/blob/b9810f7b25c80bcdc2f136d530ea742da156864c/app/geant-exporter/geant-exporter.cc#L353-L360